metis-enrichment/metis-enrichment-common/src/main/java/eu/europeana/enrichment/api/external/impl/ClientEntityResolver.java

```diff
@@ -101,7 +103,8 @@ private <I> Map<I, List<EnrichmentBase>> performBatch(boolean uriSearch, List<I>
     // TODO: 02/06/2022 This is actually bypassing the batching.. This is the selected way to perform this for now.
     for (I batchItem : batch) {
       List<EnrichmentBase> enrichmentBaseList = performItem(batchItem, uriSearch);
```

Comment:
Check and create a unit test that simulates a 400 Bad request error to the call of `ResolveByText` method in the entity-api to cover an error we experienced today seems in this class could also happen and several methods converge in this method `performItem`

```
UnknownException: Enrichment client POST call failed: https://metis-enrichment-rest-production.eanadev.org/enrich/entity/search
```

Reply:
In other words, check how the error 400 of entity-api call would affect the method `performItem`

Reply:
After talking with team this will be handled in ticket MET-2018